src/pages/ProfilePage.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import {
+    View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import Str from 'expensify-common/lib/str';
+import moment from 'moment';
+import styles from '../styles/styles';
+import Text from '../components/Text';
+import ONYXKEYS from '../ONYXKEYS';
+import Avatar from '../components/Avatar';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import {redirectToLastReport} from '../libs/actions/App';
+import HeaderGap from '../components/HeaderGap';
+import compose from '../libs/compose';
+import {version} from '../../package.json';
+
+const personalDetailsType = PropTypes.shape({
+    // Display name of the current user from their personal details
+    displayName: PropTypes.string,
+
+    // Avatar URL of the current user from their personal details
+    avatarURL: PropTypes.string,
+
+    // login of the current user from their personal details
+    login: PropTypes.string,
+
+    // pronouns of the current user from their personal details
+    pronouns: PropTypes.string,
+
+    // timezone of the current user from their personal details
+    timezone: PropTypes.string,
+});
+
+const reportType = PropTypes.shape({
+    reportID: PropTypes.number,
+    reportName: PropTypes.string,
+    participants: PropTypes.arrayOf(PropTypes.string),
+    icons: PropTypes.arrayOf(PropTypes.string),
+});
```

Comment:
I suggest create a file/folder to contain all these types based off onyx keys since they will be re-used a lot across components.

Reply:
Nice callout. I think they probably already are used in a few places and we could do this in a clean up / refactoring issue. Probably doesn't need to be done right now.